Installing fails when asciidoc is not installed

As the title says, attempting to install Plasma without assciidoc installed causes the `install` rule to fail.

Steps to recreate:
0. ensure asciidoc is not installed
1. run `make install`

Make should report an error similar to the following:
```
install -d /usr/local/bin
install -d /usr/local/share/doc/plasma

Warning: asciidoc not found, not building documentation.
--------------------------------------------------------

touch .docs_warning
cd docs/ ; for FILE in *.txt *.html; do \
        install $FILE /usr/local/share/doc/plasma; \
done
install: cannot stat '*.html': No such file or directory
make: *** [Makefile:133: install_docs] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -130,7 +130,7 @@ install_progs : install_dirs progs
 
 .PHONY: install_docs
 install_docs : install_dirs docs
-	cd docs/ ; for FILE in *.txt *.html; do \
+	cd docs/ ; for FILE in $$(find -name '*.txt' -o -name '*.html' ); do \
 		$(INSTALL) $$FILE $(DEST_DIR)$(DOCDIR); \
 	done
 	if [ -f docs/index.html ]; then \

